alpaca/data/models/bars.py

```diff
@@ -71,7 +76,10 @@ def __init__(
 
         raw_bars = raw_data
 
-        for symbol, bars in raw_bars.items():
-            parsed_bars[symbol] = [Bar(symbol, bar) for bar in bars]
+        if raw_bars is not None:
+            for symbol, bars in raw_bars.items():
+                parsed_bars[symbol] = [
+                    Bar(symbol, bar) for bar in bars if bar is not None
```

Comment:
Not important, but interestingly pylint complains about our naming, it black listed `foo, bar` words.
`bars.py:82:41: C0104: Disallowed name "bar" (disallowed-name)`